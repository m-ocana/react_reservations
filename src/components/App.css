
.app {
    font-family: sans-serif;
    text-align: center;
    border-bottom: 2px solid #666;
}

.logo {
    height: 40px;
}

.header {
    background-color: #fff;
    height: auto;
    padding: 20px;
    color: #000;
}

.intro {
    font-size: 1rem;
}
