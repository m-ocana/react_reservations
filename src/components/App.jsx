/* Feel free to edit */
import React, { Component } from 'react'
import BookingContainer from '../containers/BookingContainer'
import style from './App.css'

export default class App extends Component {
  render() {
    return (
      <div className={style.app}>
        <div className={style.header}>
          { /* TODO include, or replace heading with, logo */ }
          <h1 className={style.intro}>Table Reservations</h1>
        </div>
        <BookingContainer />
      </div>
    )
  }
}
