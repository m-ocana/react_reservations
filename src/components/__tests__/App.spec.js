import renderComponent from '../../helpers/test-component'
import App from '../App'

const renderApp = renderComponent(App)

describe('<App /> component', () => {
  describe('@renders', () => {
    it('should render App', () => {
      expect(renderApp().getTree())
        .toMatchSnapshot()
    })
  })
})
