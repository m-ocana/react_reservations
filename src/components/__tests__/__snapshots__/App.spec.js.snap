// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> component @renders should render App 1`] = `
<div
  className="app"
>
  <div
    className="header"
  >
    <h1
      className="intro"
    >
      Table Reservations
    </h1>
  </div>
  <BookingContainer />
</div>
`;
